assets/javascripts/discourse/initializers/checklist.js

```diff
@@ -100,13 +115,17 @@ export function checklistSyntax($elem, postDecorator) {
           });
 
           if (save && save.then) {
-            save.then(() => {
-              postWidget.attrs.isSaving = false;
-              postWidget.scheduleRerender();
-            });
+            save
+              .then(() => {
+                postWidget.attrs.set("isSaving", false);
```

Comment:
@jjaffeux sorry, I'm unable to find any other way to do this. https://github.com/discourse/discourse-checklist/pull/57#discussion_r887668107